src/tools/vtpipeterm/main.cpp

```diff
@@ -392,16 +392,16 @@ void handleManyEvents(const INPUT_RECORD* const inputBuffer, int cEvents)
         switch(lang)
         {
             case TEST_LANG_CYRILLIC:
-                str = "Лорем ипсум долор сит амет, пер цлита поссит ех, ат мунере фабулас петентиум сит.";

```

Comment:
Another alternative is to manually encode the hard coded strings as UTF-8 and use them in the source.   This is how I would handle this in a test that doesn't have a resource file.  Look at TEST_LANG_GOOD_POUND case below as an example.  In my day job we follow the Google Coding Standards and [here ](https://google.github.io/styleguide/cppguide.html#Non-ASCII_Characters)is what they recommend in this scenario.   (They require all of their source code be encoded as UTF-8 - which is common in cross-platform code)